src/servlet/UserDeleteServlet.java

```diff
@@ -39,18 +41,34 @@ protected void doGet(HttpServletRequest request, HttpServletResponse response)
 	 */
 	protected void doPost(HttpServletRequest request, HttpServletResponse response)
 			throws ServletException, IOException {
-		int userId = (int) (Integer.parseInt(request.getParameter("user_id")));
 
-		try {
-			UsersDAO dao = new UsersDAO();
-			dao.delete(userId);
+		HttpSession session = request.getSession(false);
+		if (session.getAttribute("login_id") == null) {
+			RequestDispatcher dispatcher = request.getRequestDispatcher("Login.jsp");
+			dispatcher.forward(request, response);
 
-			request.getRequestDispatcher("./UsersList").forward(request, response);
+		} else {
+
+			int userId = (int) (Integer.parseInt(request.getParameter("user_id")));
+
+			try {
+				int loginId = Integer.parseInt(session.getAttribute("login_id").toString());
+				if (loginId == userId) {
+					session.removeAttribute("login_id");
+					RequestDispatcher dispatcher = request.getRequestDispatcher("Login.jsp");
+					dispatcher.forward(request, response);
```

Comment:
forward後の処理は走らない前提ですが
```suggestion
					dispatcher.forward(request, response);
					return ;
```
念のためreturnで処理を終了させておきましょう。